examples/with-reasonml-todo/package.json

```diff
@@ -19,7 +19,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.4.3",
-    "bs-platform": "5.0.3",
+    "babel-types": "6.26.0",
```

Comment:
Why is this one needed?

Reply:
https://www.dropbox.com/s/49en6itmq3nwmiw/Screenshot%202020-04-05%2014.20.25.png?dl=0

Seems that it has to do with the Bucklescript babel plugin.

Reply:
The following occurs when I remove the `babel-plugin-bucklescript` from the package.json file.

```sh
[ error ] ./node_modules/next/dist/client/next-dev.js                                                                                                     │
Error: Cannot find module 'babel-plugin-bucklescript' from '/Users/nathansculli/src/scull7/next.js/examples/with-reasonml-todo'                           │
    at Array.map (<anonymous>)                                                                                                                            │
Error: Cannot find module '/Users/nathansculli/src/scull7/next.js/examples/with-reasonml-todo/.next/server/static/development/pages/_document'            │
Require stack:                                                                                                                                            │
- /Users/nathansculli/src/scull7/next.js/examples/with-reasonml-todo/node_modules/next/dist/next-server/server/load-components.js  
```

I don't know enough about the next.js dev server to understand why it's trying to load the plugin.  I've used ripgrep to search the entire node_modules folder and can't find any reference to the plugin.

Reply:
https://github.com/mike-engel/babel-plugin-bucklescript/issues/7

Seems that the babel-plugin-bucklescript has been archived and that it's got an implicit dependency.

Reply:
Seems that I'm not very smart. I didn't realize that the `babel-plugin-bucklescript` dependency was coming from the .babelrc file 🤦‍♂ .  I removed that dependency and the associated `babel-types` dependency and now all is well.